### DISCUSSION TITLE
Invalid json

### DISCUSSION THREAD

**Discussion by Treawlony** [bobokun/actualtap-py#97] [General]
I’ve smashed my head in making this work, but no matter what I keep getting this when running the shortcuts:

{
    "detail": "Expecting value: line 1 column 1 (char 0)"
}

**Comment by bruno-cobucci** [comment]
Same issue here. Have you figured it out?

EDIT: After a lot of head scratching, I found the issue. By chance, are you using a reverse proxy? I checked actual-budget logs and the issue was actually there, and not on actualtap's behalf. I later found [this issue](https://github.com/actualbudget/actual-server/issues/392), which was closed just 3 days ago, with the fix being merged to main. I hope that it fixes the issue with actualtap as well.
